app/parse-siti-payment-file/validate.js

```diff
@@ -1,11 +1,17 @@
-const checkInvoiceTotal = require('./invoice-totals')
+const getInvoiceTotal = require('./invoice-totals')
+
+const validateLineTotals = (invoiceHeaders) => {
+  return invoiceHeaders
+    .map(a => a.totalValue === getInvoiceTotal(a.lines, 'value'))
+    .every(b => b === true)
+}
 
 const validate = (invoiceBatch, invoiceHeaders) => {
   const numberOfInvoicesValid = invoiceBatch[0].numberOfInvoices === invoiceHeaders.length
-  const invoiceTotalsValid = invoiceBatch[0].batchValue === checkInvoiceTotal(invoiceHeaders, 'totalValue')
-  const invoiceLinesTotalsValid = invoiceHeaders.filter(a => a.lineTotalsValid).length === 0 ?? false
-
-  return numberOfInvoicesValid && invoiceTotalsValid && invoiceLinesTotalsValid
+  const invoiceTotalsValid = invoiceBatch[0].batchValue === getInvoiceTotal(invoiceHeaders, 'totalValue')
+  const invoiceLinesTotalsValid = validateLineTotals(invoiceHeaders)
+  const checkNumberNumberOfBatches = invoiceBatch.length === 1 ?? false
```

Comment:
Don't need the `?? false` so just: 

`const checkNumberNumberOfBatches = invoiceBatch.length === 1`